<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>402</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>322</width>
    <height>402</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>322</width>
    <height>402</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>322</width>
     <height>402</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>322</width>
     <height>402</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>322</width>
      <height>402</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>322</width>
      <height>402</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>322</width>
      <height>402</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="gameStart">
     <widget class="QLabel" name="gameTitleLabel">
      <property name="geometry">
       <rect>
        <x>47</x>
        <y>78</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ПЯТНАШКИ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="settingBtn">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>213</width>
        <height>62</height>
       </rect>
      </property>
      <property name="text">
       <string>Настройки</string>
      </property>
     </widget>
     <widget class="QPushButton" name="newGameBtn">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>213</width>
        <height>62</height>
       </rect>
      </property>
      <property name="text">
       <string>Новая игра</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="gamePlay">
     <widget class="QPushButton" name="pauseGameBtn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Пауза</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitGameBtn">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Выйти</string>
      </property>
     </widget>
     <widget class="QPushButton" name="restartGameBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Заново</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>321</width>
        <height>351</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gameBoardGridLayout"/>
     </widget>
    </widget>
    <widget class="QWidget" name="gameSetting"/>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
